src/Interfaces/IGetQueueAvailabilityOptionalParams.ts

```diff
@@ -0,0 +1,16 @@
+import InitContext from "../Model/InitContext";
+
+export default interface IGetQueueAvailabilityOptionalParams {
+  /**
+   * JWT token from the portal
+   */
+  authenticatedUserToken?: string;
+  /**
+   * Whether to get init context from browser.
+   */
+  getContext?: boolean;
+  /**
+   * Custom init context.
+   */
+  initContext?: InitContext;
```

Comment:
Is `initContext` needed?

Reply:
Yes, InitContext is required to pass on the data to execute rules from the context variables and the engagement contexts